src/Client.js

```diff
@@ -276,6 +285,14 @@ Client.prototype.request = function (options, params, callback) {
     // Add all the given parameters to the querystring.
     req = req.query(queryParams);
 
+    if (isFunction(reqCustomizer)) {
+      reqCustomizer(req, params);
+    }
+
+    if (isFunction(params._requestCustomizer)) {
+      params._requestCustomizer(req, params);
```

Comment:
I see your point. Let's leave the PR as it is right now. I'll change it in a future version if other users need to disable the global customizer.